api/api.js

```diff
@@ -0,0 +1,25 @@
+/* global ExtensionAPI:false */
+
+const { classes: Cc, interfaces: Ci, utils: Cu } = Components;
+
+this.whimsyExp = class API extends ExtensionAPI {
+  getAPI(context) {
+    return {
+      whimsyExp: {
+        setNewtabOverride: async (shouldOverride) => {      
```

Comment:
Then this would be `enableNewtab`, with future additions possibly being `enableHistory`, and `enableBookmarks`…